package.json

```diff
@@ -12,13 +12,20 @@
   "description": "Generate client-side and service-side code using swagger CLI",
   "homepage": "https://offtherecord.com",
   "author": "Off the Record, Inc.",
+  "sideEffects": [
+        "./dist/typescript/**/*",
+        "./dist/angular/**/*",
+        "./dist/otrBackendService.js",
+        "./dist/otrBackendService.min.js"
+  ],
   "scripts": {
     "download:devo": "curl https://otr-backend-service-us-devo.offtherecord.com/v2/api-docs -o api/otr-service.json",
-    "clean-source": "find dist/typescript/* -type f | grep -v  \"\\.d\\.ts\\|\\.js\" | xargs rm",
+    "clean-source": "find dist/typescript* -type f | grep -v  \"\\.d\\.ts\\|\\.js\" | xargs rm",
     "generate:devo": "npm run generate:devo-angularjs",
     "generate:devo-angularjs-ci": "npm run download:devo && gulp codegen && java -jar /opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -i api/otr-service.json -o dist/typescript -g typescript-angularjs && tsc --project tsconfig.json && npm run clean-source",
     "generate:devo-angular-ci": "npm run download:devo && gulp codegen && java -jar /opt/openapi-generator-630/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -i api/otr-service.json -o dist/angular -g typescript-angular",
     "generate:devo-angular": "npm run download:devo && gulp codegen &&  openapi-generator-cli version-manager set 6.3.0 && openapi-generator-cli generate -i api/otr-service.json -o dist/angular -g typescript-angular",
+    "generate:devo-typescript-fetch": "npm run download:devo && gulp codegen &&  openapi-generator-cli version-manager set 4.3.1 && openapi-generator-cli generate -i api/otr-service.json -o dist/typescript-fetch -g typescript-fetch --additional-properties=typescriptThreePlus=true && tsc --project tsconfig.fetch.json && npm run clean-source",
```

Comment:
we'll need to update the backend service CI pipeline to run this npm script instead.

Reply:
I was thinking that we'd have both options as we transition away from the angularJS client to the `typescript-fetch` client.

Reply:
I meant you'll need to add it so you can generate this client as well.